core/modules/views/css/views.css

```diff
@@ -56,3 +56,7 @@
 .view .progress-disabled {
   float: none;
 }
+
+.views-grid .views-row {
+  display: flex;
```

Comment:
Needs to be run through autoprefixer, check to see if there's a plugin for your code editor of choice, or go to autoprefixer.github.io